content/400-reference/300-database-reference/01-database-features.mdx

```diff
@@ -123,18 +123,18 @@ This section describes which database features exist on the NoSQL databases that
-| Arrays                                    | ✔️                  |                                                                                                                                                                                                          |
-| Enums                                     | ✔️                  | Implemented at Prisma level                                                                                                                                                                              |
-| Native database types                     | ✔️                  | See: [Field mapping reference](/reference/api-reference/prisma-schema-reference#model-field-scalar-types)                                                                                                |
-| JSON support                              | ✔️                  | Advanced `Json` field filtering is supported by [Postgres and MySQL since 2.26.0](/concepts/components/prisma-client/working-with-fields/working-with-json-fields#filtering-on-a-json-field)             |
+| Feature                                   | Supported by Prisma | Notes                                                                                                                                                                                        |
+| :---------------------------------------- | :------------------ | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Embedded documents                        | ✔️                  |                                                                                                                                                                                              |
+| Transactions                              | ✔️                  |                                                                                                                                                                                              |
+| Indexes                                   | ✔️ with caveats     | Indexes can only be introspected if the field they refer to includes at least some data.                                                                                                     |
+| Autoincrementing IDs                      | No                  |                                                                                                                                                                                              |
+| Compound IDs                              | No                  | MongoDB does not support composite IDs (`@@id`)                                                                                                                                              |
```

Comment:
Both are not supported on MongoDB, but the table is called "common MongoDB features and describes the level of support offered by Prisma" - that might be misleading or at least confusing.

Reply:
I see the table as providing info about common features, and these two features are worth calling out and clearly noting that we don't support. In other words, the table is I think) meant to provide info on all common features, not just the ones we support.

Reply:
Then the description above the table is wrong. Then it is not "common MongoDB features and describes the level of support offered by Prisma".